content/sensu-go/5.17/reference/agent.md

```diff
@@ -997,6 +1001,19 @@ sensu-agent start --keepalive-critical-timeout 300
 # /etc/sensu/agent.yml example
 keepalive-critical-timeout: 300{{< /highlight >}}
 
+<a name="keepalive-handlers-flag"></a>
+
+| keepalive-handlers |      |
+--------------------|------
+description         | [Keepalive event handlers][52] to use for the entity, specified in a comma-delimited list. You can specify any configured handler and invoke the `keepalive-handlers` flag multiple times. If keepalive handlers are not specified, the Sensu backend will use the default keepalive handler (sending an alert to your dashboard).
```

Comment:
> will use the default keepalive handler (sending an alert to your dashboard).

Actually keepalive events will attempt to use a default handler named `keepalive`, which does not exist on fresh installations, but it will indeed end up as an event in sensuctl & the web UI.